connectors/pme/pme_ui_dev.js

```diff
@@ -244,12 +241,12 @@ var SaveToFlow = (function() {
 
 	function tracking(doc, tracking) {
 		try {
-			doc.getElementById("stf_track_found").value = tracking.found;
-			doc.getElementById("stf_track_selected").value = tracking.selected;
-			doc.getElementById("stf_track_url").value = tracking.url;
-			doc.getElementById("stf_track_citation").value = tracking.citation;
-			doc.getElementById("stf_track_modified").value = JSON.stringify(tracking.modified);
-			//doc.getElementById("stf_tracking_form").submit();
+			doc.getElementById("s2r-track_found").value = tracking.found;
+			doc.getElementById("s2r-track_selected").value = tracking.selected;
+			doc.getElementById("s2r-track_url").value = tracking.url;
+			doc.getElementById("s2r-track_citation").value = tracking.citation;
+			doc.getElementById("s2r-track_modified").value = JSON.stringify(tracking.modified);
+			//doc.getElementById("s2r-tracking_form").submit();
```

Comment:
I get the feeling that this shouldn't be commented out (though I see it already was)